stratum/docs/extenting_stratum_bcm.md

```diff
@@ -1,22 +1,27 @@
-# How to add a Feature to Stratum (BCM)
+# How to Add a Dataplane Feature to Stratum (BCM)
 
 This guide describes how to add a new feature to Stratum BCM on the example of MPLS.
 
 ## 0. Prepare Testbed
 
-Developing on real hardware instead of simulators
+Stratum does not support simulators for the ASIC, therefore developing on real
```

Comment:
```suggestion
Stratum BCM does not support simulators for the ASIC, therefore developing on real
```

We can run Stratum on top of tofino-model.

Reply:
Should we also mention that `stratum/hal/bin/bcm/sim` is invalid for now in this document?

Reply:
As far as we know it does not work and we have no plans to fix it. I'd not even mention it here.